src/include/ipc/header.h

```diff
@@ -86,10 +86,12 @@
  *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
  *     | Core                                                          |
  *     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- *     | Reserved                                                      |
+ *     | ext_data_length                                               |
```

Comment:
I doubt `ext_data_length` will ever need 32bits / 4GB, will it? If not then some bits should be kept reserved.

PS:  thanks to little-endianness these bits could easily be reserved again even after this gets merged.

Reply:
Lets use full extended size - anything new here will be added to extended data and this keeps it simple prior to IPC2.0

Reply:
@marc-hb I agree with Liam. 32bit fields are faster to parse on the DSP and if we need to add something more, we now have the extended data defined, so you can add any new fields at end of extended data.